<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortPushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>LessThan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Show full info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowAll</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>96</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>code</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>price</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lessThanLineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
